.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,45 @@
+name: Building Docker images
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  release:
+    types:
+      - created
+
+jobs:
+  build_docker:
+    name: Build docker image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Build and push Docker base image
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          repository: hellofreshtech/kangal-jmeter
+          path: docker/jmeter-base/.
+          tags: latest
```

Comment:
I think we should build tags with version on release being created and use fixed versions as tags, so base image and master/worker should have the same tag. Latest also can be - e.g. the latest merged to master.

You can find example for using tags from ref @ https://github.com/vgarvardt/rklotz/blob/master/.github/workflows/main.yml